### DISCUSSION TITLE
Issues joining the cncf-lfx-mentees-22-term3 group

### DISCUSSION THREAD

**Discussion by AllMight2099** [cncf/mentoring#715] [General]
Hi there!
I'm Nishanth, an LFX mentee under KubeArmor for the 2022 Fall Term. I received an email from CNCF with an invite to join the cncf-lfx-mentees-22-term3 group. But when I do click on the link to accept the invite, I get a ```This Group Does Not Exist``` error a. I've also tried replying via the same email and emailing the owners group, but I get an error from the Mail Delivery Subsystem with an error that says "The email wasn't sent because the address wasn't found". 

I'm posting this as I couldn't find any other way of getting in touch with personnel from the CNCF, so I'd be grateful if someone could help me with this

**Comment by AllMight2099** [comment]
cc: @nate-double-u

**Reply by nate-double-u** [reply]
Sorry about that, i've resent the invite.
